generated/nifgen/nifgen.proto

```diff
@@ -165,27 +165,39 @@ service NiFgen {
 
 enum NiFgenAttributes {
   NIFGEN_ATTRIBUTE_UNSPECIFIED = 0;
+  NIFGEN_ATTRIBUTE_RANGE_CHECK = 1050002;
+  NIFGEN_ATTRIBUTE_QUERY_INSTRUMENT_STATUS = 1050003;
+  NIFGEN_ATTRIBUTE_CACHE = 1050004;
   NIFGEN_ATTRIBUTE_SIMULATE = 1050005;
+  NIFGEN_ATTRIBUTE_RECORD_COERCIONS = 1050006;
   NIFGEN_ATTRIBUTE_DRIVER_SETUP = 1050007;
-  NIFGEN_ATTRIBUTE_NUM_CHANNELS = 1050203;
```

Comment:
Why was this attribute removed?

Reply:
NUM_CHANNELS is renamed to CHANNEL_COUNT to match header